docs/src/styles/docs/sidebar.scss

```diff
@@ -1,55 +1,160 @@
+@import './_variables.scss';
+
+
+@keyframes showSidebar {
+  0% {
+    transform: translate(-100%);
+  }
+  100% {
+    transform: translate(0);
+  }
+}
+
+@keyframes showOverlay {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
 .docs-sidebar {
-  display: none;
+  display: block;
   
-  @media (min-width: 62rem) {
-    display: block;
+  &.collapsed {
+    display: none;
+    @media (min-width: $breakpoint-large) {
+      display: block;
```

Comment:
Looks like on the home page this is allowing you to tab through "invisible" items. Should it be scoped to just non-homepage pages somehow? (we have a .docs-home but not a .docs-inner or whatever you might call a non-homepage I don't think)

Reply:
Ok this should be fixed now!